sandy/core/cov.py

```diff
@@ -599,14 +621,21 @@ def sampling(self, nsmp, seed=None, rows=None, pdf='normal',
         0    2.04904e-03
         1   -3.31541e-03
         dtype: float64
+
+        >>> sandy.CategoryCov([[1, -2],[-2, 3]]).sampling(1000000, seed=11, pdf='lognormal', tolerance=0, relative=True).data.mean(axis=0)
+        0   1.00011e+00
+        1   9.99829e-01
+        dtype: float64
         """
         dim = self.data.shape[0]
-        y = sample_distribution(dim, nsmp, seed=seed, pdf=pdf)
+        y = sample_distribution(dim, nsmp, seed=seed, pdf=pdf) - 1
         y = sps.csc_matrix(y)
         L = sps.csr_matrix(self.get_L(rows=rows, tolerance=tolerance))
         samples = L.dot(y).toarray()
         if relative:
-            samples += 1  # mean=1, to be multiplied by best estimate
+            samples += 1  # mean=1, to be multiplied by the best estimate
+        elif pdf == 'lognormal':
+            logging.warning("LogNormal distribution should not have mean=0")

```

Comment:
Is this a warning or an error? This `if` condition should be clarified

Reply:
My idea is to leave open the option to create samples centered in zero with any distribution (which results in the translation of the lognormal distribution with mean 1, which is lognormal only in its 'shape'). This is to allow the user to sample absolute covariance matrices with lognormal distributions as well. The warning is just to make the user aware of what are the risks involved in this procedure. In my opinion, SANDY default sampling algorithm should always use relative covariances, bypassing this issue.